src/ray/common/ray_config_def.h

```diff
@@ -311,6 +311,12 @@ RAY_CONFIG(int, worker_niceness, 15)
 RAY_CONFIG(int64_t, redis_db_connect_retries, 600)
 RAY_CONFIG(int64_t, redis_db_connect_wait_milliseconds, 100)
 
+/// Number of retries for a redis request failure.
+RAY_CONFIG(int64_t, num_redis_request_retries, 3)
+
+/// Interval between two retries.
```

Comment:
would 300ms retry catch the normal failure cases? i'd expect exponential back-offs with longer delays to work.

Reply:
I can update it to that. I was thinking that for the short term support of this, we only cover the short duration failure.
Here reconnection won't be supported, so only certain network failure can be supported.